lib/Backend/Lower.cpp

```diff
@@ -3993,7 +3993,22 @@ IR::JnHelperMethod GetArrayAllocMemHelper<Js::JavascriptNativeFloatArray>()
 
 template <typename ArrayType>
 IR::RegOpnd *
-Lowerer::GenerateArrayAlloc(IR::Instr *instr, uint32 * psize, Js::ArrayCallSiteInfo * arrayInfo, bool * pIsHeadSegmentZeroed, bool isArrayObjCtor /* = false */)
+Lowerer::GenerateArrayLiteralsAlloc(IR::Instr *instr, uint32 * psize, Js::ArrayCallSiteInfo * arrayInfo, bool * pIsHeadSegmentZeroed)
```

Comment:
>GenerateArrayLiteralsAlloc [](start = 9, length = 26)

ArrayLiterals means something else and i guess it doesn't suit this scenario.
Moreover, why do you need 2 wrappers ? All you need is one extra argument in the existing and last two arguments can be default params.